plutus-core/cost-model/data/models.R

```diff
@@ -89,6 +89,16 @@ get.bench.data <- function(path) {
         mutate(across(c("Mean", "MeanLB", "MeanUB", "Stddev", "StddevLB", "StddevUB"), seconds.to.milliseconds))
 }
 
+filter.and.check.nonempty <- function (frame, name) {
```

Comment:
This helps with error checking.  When the names in the CSV file changed I was getting cryptic errors with no context because data was missing.  This will check for that case and tell you what's missing.